scripts/setup-env-files.js

```diff
@@ -15,14 +16,10 @@ const prePopulateEnv = (folders, folderBasePath, exampleEnvFilePath = 'src/.exam
 };
 
 (async () => {
-  const apps = ['api', 'ws', 'worker'];
   const appsBasePath = `${__dirname}/../apps`;
-
   console.log('----------------------------------------');
-  console.log('Pre-populating .env files from .example.env');
-
-  prePopulateEnv(apps, appsBasePath);
-
+  prePopulateEnv(['api', 'ws', 'worker'], appsBasePath);
+  prePopulateEnv(['web', 'widget'], appsBasePath, '.example.env', '.env');
```

Comment:
web and widget don't have their `.env` files in `/src` but in the parent directory, so this adjustment is needed

Reply:
😮‍💨